zeroconf/test.py

```diff
@@ -16,8 +17,11 @@
 
 from nose.plugins.attrib import attr
 
-import zeroconf as r
-from zeroconf import (
+# ensure I can find this package even when it hasn't been installed (for development purposes)
+sys.path.insert(0, '..')
+
+import zeroconf as r  # noqa: E402
+from zeroconf import (  # noqa: E402
```

Comment:
Those `# noqa` pragmas are out of scope of this pull request.

Reply:
Understood